<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Dostępny</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zaraz wracam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Niewidoczny</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Niedostępny</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionUstawienia"/>
    <addaction name="actionDodaj_kontakt"/>
    <addaction name="separator"/>
    <addaction name="actionZako_cz"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUstawienia">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionDodaj_kontakt">
   <property name="text">
    <string>Dodaj kontakt</string>
   </property>
  </action>
  <action name="actionZako_cz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
